src/lib/apify_command.js

```diff
@@ -10,10 +11,23 @@ class ApifyCommand extends Command {
     parse(cmd) {
         const { flags, args } = super.parse(cmd);
         const parsedFlags = argsToCamelCase(flags);
-        return {
-            flags: parsedFlags,
-            args,
+        // Initialize telemetry data, all attributes are tracked in the finally method.
+        this.telemetryData = {
+            command: cmd.id,
+            flagsUsed: Object.keys(parsedFlags),
         };
```

Comment:
The `parse` method is not called in every command, like `apify actor:get-input`, so the `this.telemetryData` is not set and the telemetry collection ends with an error.

We should also add an `init` method, and create the `this.telemetryData` with the command ID in there, and here in the `parse` method only add there the `flagsUsed`.

https://github.com/oclif/core/blob/a2efc040855969678a6aaf14d3ceb1d25d7a8411/src/command.ts#L299